src/gt4py/next/ffront/type_info.py

```diff
@@ -17,7 +17,7 @@
 
 import gt4py.next.ffront.type_specifications as ts_ffront
 import gt4py.next.type_system.type_specifications as ts
-from gt4py.next.common import Dimension
+from gt4py.next.common import Dimension, promote_dims
```

Comment:
```suggestion
from gt4py.next import common
```
to follow our style guide (please update all usages of Dimension, too)